pkg/block/block.go

```diff
@@ -95,3 +101,59 @@ func renameFile(from, to string) error {
 	}
 	return pdir.Close()
 }
+
+// DownloadDir downloads directory that is mean to be block directory (ends with ULID and contains block data).
+func DownloadDir(ctx context.Context, bucket objstore.Bucket, blockDir, dst string) error {
+	if _, err := ulid.Parse(path.Base(blockDir)); err != nil {
+		return errors.Errorf("given directory is not block directory %s", blockDir)
+	}
+
+	if err := objstore.DownloadDir(ctx, bucket, blockDir, dst); err != nil {
+		return err
+	}
+
+	chunksDir := filepath.Join(dst, "chunks")
+	_, err := os.Stat(chunksDir)
+	if os.IsNotExist(err) {
```

Comment:
Do we really need this? Just `os.MkdirAll` should work whether it exists already or not.

Reply:
Hm.. Just it is more explicit that we do that only if not exists

Reply:
Yea, guess it doesn't matter.